doc/ci.md

```diff
@@ -0,0 +1,44 @@
+# Sistemas de integración continua
+
+## Criterios
+
+- Gratuidad del servicio: el servicio debe ser gratuito.
+- Integración con Docker y Github: es estrictamente necesario que se pueda integrar con Github Checks. Además, se valorará positivamente si también permite integración con Docker.
+- Fiabilidad: como en todas las decisiones que hemos tomado hasta el momento, el sistema debe tener actualizaciones recientes. 
+- Simplicidad: con esto básicamente me refiero a que se puede usar el servicio de manera online, y no requiere ninguna instalación local.
+
+## Opciones
+
+### Jenkins
```

Comment:
De verdad, no me pongáis Jenkins más. Si dijo @pluque01 delante tuyo que había que descargarse un fichero y ejecutarlo localmente.